File: sdkwork-backend-sdk-typescript/src/types/plus-api-security-policy-vo.ts
/** API安全策略VO */
export interface PlusApiSecurityPolicyVO {
  createdAt?: string;
  updatedAt?: string;
  id?: number;
  uuid?: string;
  policyCode?: string;
  apiType?: string;
  pathPattern?: string;
  httpMethod?: string;
  matchMode?: 'EXACT' | 'PREFIX' | 'ANT';
  authMode?: 'NONE' | 'API_KEY' | 'AUTH_TOKEN' | 'ACCESS_TOKEN' | 'AUTHENTICATED';
  allowAnonymous?: boolean;
  requiredRoles?: string;
  requiredPermissions?: string;
  priority?: number;
  enabled?: boolean;
  description?: string;
}

File: sdkwork-backend-sdk-go/types/plus_api_security_policy_vo.go
package types

// API安全策略VO
type PlusApiSecurityPolicyVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	PolicyCode string `json:"policyCode"`
	ApiType string `json:"apiType"`
	PathPattern string `json:"pathPattern"`
	HttpMethod string `json:"httpMethod"`
	MatchMode string `json:"matchMode"`
	AuthMode string `json:"authMode"`
	AllowAnonymous bool `json:"allowAnonymous"`
	RequiredRoles string `json:"requiredRoles"`
	RequiredPermissions string `json:"requiredPermissions"`
	Priority int `json:"priority"`
	Enabled bool `json:"enabled"`
	Description string `json:"description"`
}
